### PR TITLE
docs: better semantic reading CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by goobric**
Changed line 94, to give a fuller explanation of the reason to 'run tests'
Thank you Emma Dawson @emmalearnscode , for the wording/phrasing for line 94

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1233"></a>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ After making changes make sure that tests passes
 npm run start
 ```
 
-**2.** Run the tests by running this command
+**2.** <!--Run the tests by running this command--> The project runs automated tests when creating a PR. It tests components of the App. If you wish to run tests manually use the command ‘npm run cypress:run’ . If you need help to fix failing tests please comment on the PR asking for help.
```
Comment (schmelto) [inline CONTRIBUTING.md:94]:
```suggestion
**2.** The project runs automated tests when creating a PR. It tests components of the App. If you wish to run tests manually use the command ‘npm run cypress:run’ . If you need help to fix failing tests please comment on the PR asking for help.
```

**Review by schmelto — COMMENTED** [review]
could we remove this comment?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Review by emmalearnscode — COMMENTED** [review]
I'm wondering if instruction 1 is irrelevant in this section, especially if we make the text clearer like Mikal is suggesting. npm start doesn't really have anything to do with running tests and is not necessary to run the test command. Should we remove it? Otherwise we need to make it clearer what the purpose of doing instruction 1 is.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Comment by goobric** [comment]
I have closed this PR. Thank you for your support.
For future reference, I can invite collaborators by following these instructions in this GH doc?
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository

**Comment by goobric** [comment]
Do I now 'Delete branch' ?

**Comment by emmalearnscode** [comment]
Since this is not resolved do you mind if I open a new issue asking someone else to replace the text @goobric ?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Comment by goobric** [comment]
I selected 'Deleted Branch' and deleted it.
Now the message is asking "If you wish, you can delete this fork of EddieHubCommunity/LinkFree" in the settings!
I will delete the fork.
Very confused; as I am completely unaware if the proposed changes and have been agreed and implemented or not?!? :-/

**Comment by emmalearnscode** [comment]
@goobric this line still needs to be added as it didn’t get merged: The project runs automated tests when creating a PR. It tests components of the App. If you wish to run tests manually use the command ‘npm run cypress:run’ . If you need help to fix failing tests please comment on the PR asking for help.

**Pull request closed without merging**